lib/src/Uncertainty/Bayesian/RandomWalkMetropolisHastings.cxx

```diff
@@ -146,6 +149,17 @@ Distribution RandomWalkMetropolisHastings::getProposal() const
   return proposal_;
 }
 
```

Comment:
The line before the `set` method should give a small comment of its purpose, see e.g. [Rice.cxx#L223](https://github.com/openturns/openturns/blob/233e5ddd33c6c1f37b49482a9ca30f4d2e520e25/lib/src/Uncertainty/Distribution/Rice.cxx#L223):
```suggestion

/* Burning accessor */
```